remember_remember.py

```diff
@@ -0,0 +1,14 @@
+# write function that receives a picture and finds the secret message
+
+from PIL import Image
+
+PATH = 'code.png'
+
+message = ''
+with Image.open(PATH) as img:
```

Comment:
Wrap your code in functions, this code is not modular when you implement it in the outer scope